src/backup/default_backup_predicate.py

```diff
@@ -49,6 +48,14 @@ def __is_table_backup_up_to_date(self, big_query_table_metadata, last_backup):
         if source_table_last_modified_time > last_backup.last_modified:
             logging.info("Backup time is older than table metadata")
             return False
+
+        source_table_num_bytes = big_query_table_metadata.table_size_in_bytes()
+        if source_table_num_bytes != last_backup.numBytes:
+            logging.info("Backup size is different than source. "
```

Comment:
As in theory this should never happen, maybe we should log also source/backup last update time, so it will be easier to analyse based on logs?

Reply:
It's already logged at the begining of __is_table_backup_up_to_date method. 
So informations are already in place.